nf_core/schema.py

```diff
@@ -513,12 +514,15 @@ def schema_to_markdown(self, columns):
                             out += f"<details><summary>Help</summary><small>{help_txt}</small></details>"
                     elif column == "type":
                         out += f"| `{param.get('type', '')}` "
+                    elif column == "required":
+                        out += f"| {p_key in required or ''} "
                     else:
                         out += f"| {param.get(column, '')} "
                 out += "|\n"
 
         # Top-level ungrouped parameters
         if len(self.schema.get("properties", {})) > 0:
+            required = self.schema.get("required", [])
```

Comment:
Should this one not also be added to `out` below, the same as we above?

Reply:
D'oh

Reply:
I've refactored the table generating code which was the same for the if/else block into its own method. I realised that I hadn't captured the `<br>` changes from the previous PR in the top-level properties section either 🙈